homeassistant/components/openhome/media_player.py

```diff
@@ -124,6 +126,13 @@ def turn_off(self):
         """Put device in standby."""
         self._device.SetStandby(True)
 
+    def play_media(self, media_type, media_id, **kwargs):
+        """Send the play_media command to the media player."""
+        track_details = {}
+        track_details["title"] = "Home Assistant"
+        track_details["uri"] = media_id
+        self._device.PlayMedia(track_details)
```

Comment:
What happens if the `media_type` is a movie or a video?

Reply:
That’s a good question. In this case it would send on the url to the device which may or may not be able to play it. While it *might* work, I think this should restrict to audio only. Are the media types defined in a constant I can use?

Reply:
Thanks, that was my thought as well. Yes, there is the following constant for this case:
https://github.com/home-assistant/home-assistant/blob/7685c76b9b49408dea07c947c2eb54fe1f01f92f/homeassistant/components/media_player/const.py#L32
However, you should also consider the other constants, for example, this one:
https://github.com/home-assistant/home-assistant/blob/7685c76b9b49408dea07c947c2eb54fe1f01f92f/homeassistant/components/media_player/const.py#L38

Reply:
Just pushed a new version which restricts to `music` only. `playlist` would require more support in the device library in use.